tests/unit/components/ont/OntTubeSelectedList.spec.js

```diff
@@ -28,8 +28,11 @@ describe('OntTubeSelectedList', () => {
     })
 
     it('contains the correct data', async () => {
-      expect(wrapper.find('tbody').findAll('tr').length).toEqual(1)
-      expect(wrapper.find('.b-table-empty-row').text()).toEqual('No tubes selected')
+      expect(wrapper.find('tbody').findAll('tr').length).toEqual(0)
+      /*TODO-:The empty-text feature need to be implemented in custon-table.
```

Comment:
Any thoughts on when this would be done?  I tend to have a rule that TODO statements need to name the issue number they will be addressed in and that issue should have an acceptance criteria to resolve the TODO statements they are tagged by.  They are easier to find then with a project wide search.

Reply:
Actually, I have implemented this now. Pushed in as well